docs/DevelopmentGuide.md

```
@@ -182,6 +182,16 @@ Due to technical limitations, the workflow can only post pull request comments i
 If you are working on a fork, you can still see the clang-tidy results either in the GitHub Actions logs,
 on the workflow summary page, or in the "Files changed" tab of the pull request.
 
+To get that report before pushing, the `cpp-lint` nox session runs the very same check locally.
+
+```console
+$ nox -s cpp-lint
```

markdownlint-cli2 (0.23.2)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)